WooCommerce/Classes/ViewRelated/Products/Variations/ProductVariationsViewController.swift

```diff
@@ -571,7 +571,9 @@ private extension ProductVariationsViewController {
         actionSheet.addDefaultActionWithTitle(ActionSheetStrings.bulkUpdate) { [weak self] _ in
             guard let self = self else { return }
 
-            let viewModel = BulkUpdateViewModel(siteID: self.siteID, productID: self.productID)
+            let viewModel = BulkUpdateViewModel(siteID: self.siteID, productID: self.productID) { [weak self] in
+                self?.dismiss(animated: true)
+            }
```

Comment:
Nit: I think we should not omit the closure name here, as a reader I'm not sure what does the trailing closure is about, cancel? success? error?

Reply:
I agree giving more context here makes it more clear and readable. I fixed it!